src/cstart.sh

```diff
@@ -35,17 +36,21 @@ fi
 echo "ip $ip"
 
 # port
-if [ -z "$port" ]; then
+if [ -e port ] ; then
+    port=`cat port`
+else
     while [ true ]
     do
         port="$(echo $(( RANDOM % 1000 + 40000 )))"
-        ss -a -n | grep LISTEN | grep "${ip}:${port} " 1>/dev/null 2>&1 || break
+        ss -a -n | grep LISTEN | grep "${ip}:${port} " 2>&1 >/dev/null || break
     done
+    echo $port > port
 fi
-echo "port $port"
+echo "mon port $port"
+
 
 # make sure we have an image
-if ! $runtime image inspect $image_base:$shortid 2>/dev/null; then
+if ! sudo $runtime image inspect $image_base:$shortid 1>/dev/null 2>/dev/null; then
```

Comment:
nit, could be more consistent on how to redirect stdout and stderr to `/dev/null`, i.e. to use 
```sh
sudo $runtime image inspect $image_base:$shortid 2>&1 >/dev/null
```